Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/TheAlgorithms/index/) for The Algorithms Website/Index.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/TheAlgorithms/-/index/horizontal-auto.svg)

AnshulMalik commented:
Nice work! Few things to mention though.

This is a perfect example to use css grids instead of tables :) And we'd ideally like to display vertically centered to the icon(it goes to the bottom of the icon currently).
And we'll probably won't need the border of the table.

Well done :)

co16353sidak commented:
@AnshulMalik 
We could use grids but that could cause unnecessary complications to the css, or if you agree on using a pre-defined grid library we could call it via a cdn which will only load the modules called in the code. What do you say. 😃 
P.S: I know the only one that comes to mind is bootstrap but there are far more better and lighter grid-only css libraries out there for us to choose from. 🔥

AnshulMalik commented:
Please have a look at this article https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout

We don't need those libraries at all :)

co16353sidak commented:
@AnshulMalik 
have made the required changes, please check.

Updated Screenshot:
<img width="1392" alt="Screenshot 2019-05-16 at 11 31 14 PM" src="https://user-images.githubusercontent.com/35738138/57876506-199e9800-7833-11e9-972b-18636cdbdae8.png">

AnshulMalik commented:
This looks awesome :) 
Just one thought, the language icon and text look like they are separated by a large space and the text is closer to the other language's icon. Can we make it such that the language name is closer to it's icon? this would then be perfect :)

co16353sidak commented:
@AnshulMalik made the requested changes. 👍

AnshulMalik commented:
This is perfect :) And ready to merge :)
It would be cherry on cake, if we can make do something on hover on any language or it's icon.
May be slight darker background and make this whole box clickable. 

But that can be done in another PR, Love this :)

co16353sidak commented:
> It would be cherry on cake, if we can make do something on hover on any language or it's icon.

@AnshulMalik would love to work on this as well, look out for another issue and PR soon 😄

maltejur committed:
Added translation using Weblate (Spanish)

maltejur committed:
Translated using Weblate (Spanish)

Currently translated at 25.0% (1 of 4 strings)

Translation: The Algorithms Website/Index
Translate-URL: https://hosted.weblate.org/projects/TheAlgorithms/index/es/
